PlanNEU midsemester demo

sanjana-singhania opened the pull request:
do not merge, just making a PR for the vercel deployment

megandouli reviewed (approved):
:chungus: PR! quick question, can we add a bit more space between "SearchNEU" and the box containing the search bar and dropdown? They look a little close on mobile rn

bugsalexander commented:
> :chungus: PR! quick question, can we add a bit more space between "SearchNEU" and the box containing the search bar and dropdown? They look a little close on mobile rn

Yep, good call! Fixed:

![Screen Shot 2021-03-08 at 10 56 19 PM](https://user-images.githubusercontent.com/22948432/110416326-8797fd00-8061-11eb-9e23-0826fa233156.png)

jpnortonwastaken committed:
Add filtering

jpnortonwastaken committed:
Add filter frontend for testing

ItsEricSun committed:
add side filter panel

ItsEricSun committed:
Merge branch 'main' into filtering

ItsEricSun committed:
add scheduler flag to page header

aarowz committed:
Add NUPath filtering to schedule generator

- Add nupaths filter type to ScheduleFilters
- Fetch course NUPath data from database with SQL aggregation
- Add NUPath selection UI in FilterPanel with checkboxes
- Display active NUPath filters in SchedulerView
- Filter schedules to show only those fulfilling all selected NUPaths

jpnortonwastaken committed:
Add calendar view

jpnortonwastaken committed:
Add little fixes

sanjana-singhania committed:
enable feature flag
